modules/facilities_api/app/controllers/facilities_api/v1/va_controller.rb

```diff
@@ -51,5 +56,17 @@ def serializer
     def resource_path(options)
       v1_va_index_url(options)
     end
+
+    def mobile_api
+      FacilitiesApi::V1::MobileCovid::Client.new
+    end
+
+    def mobile_api_get_by_id(id)
```

Comment:
Using a more self-documenting name here would improve code readablility, e.g. `get_ covid_online_scheduling_available_from_mobile_api`.

Reply:
its aliasing `mobile_api.direct_booking_eligibility_criteria_by_id(id).covid_online_scheduling_available?`

Reply:
Not sure I follow but this is a minor concern.

Reply:
I was just making a shorter method name since the other one is descriptive. I can get rid of it but it's just an alias to a long method chain.